person.rb

```diff
@@ -2,14 +2,15 @@
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :person_rentals
```

Comment:
- Let's make the person_rentals to `rentals` to be consistent. a rental is a rental and someone would eventually be renting the book. so making it `rentals` is more readable.

Reply:
Done!